src/index.html

```diff
@@ -1,6 +1,18 @@
 <!doctype html>
 <html lang="en">
   <head>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
```

Comment:
I don't think we need so many fonts if we only use 3.

```suggestion
      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
      rel="stylesheet"
    />
```

Reply:
yep, that's my problem, I was in a hurry and forgot about run deploy, and thank you for your feedback.